src/main/java/com/passbook/sparkeighteen/controller/TransactionController.java

```diff
@@ -0,0 +1,33 @@
+package com.passbook.sparkeighteen.controller;
+
+import com.passbook.sparkeighteen.peristence.POJO.TransactionRequest;
+import com.passbook.sparkeighteen.peristence.POJO.TransactionResponse;
+import com.passbook.sparkeighteen.peristence.POJO.TransactionType;
+import com.passbook.sparkeighteen.service.TransactionService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RequestMapping("/api/v1/user")
+@RestController
+public class TransactionController {
```

Comment:
How will i see the transaction of that user ?

Reply:
Updated, As perviously discussed need this to be closed